packages/modals/src/utils/useCombinedRefs.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
This should exist; `import { useCombinedRefs } from '@zendeskgarden/container-utilities'`.